CMakeLists.txt

```diff
@@ -45,7 +45,9 @@ find_ros1_interface_packages(ros1_message_packages)
 set(prefixed_ros1_message_packages "")
 foreach(ros1_message_package ${ros1_message_packages})
   find_ros1_package(${ros1_message_package} REQUIRED)
```

Comment:
I think this should also be done within the `if(NOT ...)` call - we don't need to find the package if we don't use it.

Reply:
done in https://github.com/ros2/ros1_bridge/pull/152/commits/6c953ce386ac8c6785e3c03472ba9d0b6a98bfe8